routes/web.php

```diff
@@ -236,13 +236,20 @@
         Route::resource('beatmapsets', '\App\Http\Controllers\BeatmapsetsController', ['only' => ['show']]);
 
         //  GET /api/v2/me
-        Route::get('me', 'UsersController@me');
+        Route::get('me', '\App\Http\Controllers\UsersController@me');
```

Comment:
at this rate might as well remove the `API` prefix from route group? <_<

Reply:
soon™